modules/theme/src/themes/default/collections/form.overrides

```diff
@@ -180,3 +180,14 @@
         padding-left: 2.75rem;
     }
 }
+
+/*-----------------
+  Label styles
+------------------*/
+
+.oxygen-form-control-label {
```

Comment:
Lets update this in `@oxygen-ui` side and bring in. Can be done in a followup PR.

Reply:
Ack. All the changes required are being tracked in this sheet https://docs.google.com/spreadsheets/d/1_7ieNvDRK6aEvSPkErT-hhfh-R5dEVuKV6kU1W_h1Hs/edit?usp=sharing

We will address them asap.